extension/lib/shopify/AdminClient.js

```diff
@@ -10,32 +10,17 @@ class ShopifyAdminClient extends ShopifyClient {
     super({shopName: shopName, accessToken: adminAccessToken})
   }
 
-  /**
-   * @param {number} id
-   * @return {Promise<number>}
-   */
-  async getProductCountByCollectionId (id) {
-    return this.product.count({collection_id: id})
-  }
-
   /**
    * @return {Promise<ShopifyAccessTokenObject>}
    */
   async createStorefrontAccessToken () {
     return this.storefrontAccessToken.create({title: STOREFRONT_ACCESS_TOKEN_TITLE})
   }
 
-  /**
-   * @return {Promise<ShopifyAccessTokenObject[]>}
-   */
-  async getStorefrontAccessTokens () {
-    return this.storefrontAccessToken.list()
-  }
-
   /**
    * @return {Promise<string>}
    */
-  async getStorefrontAccessToken () {
+  async getFirstApplicableStorefrontAccessToken () {
```

Comment:
This seems unused and I couldn't find a reference by searching the project. Can this be removed?

Reply:
It can, I was wondering should I leave it or could we have any use of it later.

Reply:
Removed.